ui/webui/test/check-language

```diff
@@ -57,10 +57,17 @@ class TestLanguage(MachineCase):
         b.wait_visible("#English--English-")
         b.wait_visible("#Deutsch--German-")
 
-        # Check that the language menu shows filtered menu entries
+        # Check filtering on English names
+        b.set_input_text(f"#{i.welcome_id}-menu-toggle-select-typeahead", "cze")
+        b.wait_not_present("#Deutsch--German-")
+        b.wait_not_present("#English--English-")
+        b.wait_visible(f"#{i.welcome_id}-option-cs_CZ")
+
+        # Check filtering on native names
         b.set_input_text(f"#{i.welcome_id}-menu-toggle-select-typeahead", "Deutsch")
-        b.wait_visible("#Deutsch--German-")
         b.wait_not_present("#English--English-")
+        b.wait_not_present("#installation-language-option-cs_CZ")
```

Comment:
Sorry, can you please parameterize this as well as you did for above?

Reply:
argh sorry :/ of course!

Reply:
done